src/pages/iou/steps/MoneyRequestAmountForm.js

```diff
@@ -76,8 +70,6 @@ function MoneyRequestAmountForm({amount, currency, title, isEditing, onBackButto
         end: selectedAmountAsString.length,
     });
 
-    const textInput = useRef(null);
```

Comment:
I don't think we should remove the internal ref and relay just on `forwardedRef`. Let's keep our ref and just forward it to `forwardedRef`.

The ref function would be similar to BaseTextInput

```js
                                ref={(ref) => {
                                    if (typeof props.innerRef === 'function') {
                                        props.innerRef(ref);
                                    } else if (props.innerRef && _.has(props.innerRef, 'current')) {
                                        // eslint-disable-next-line no-param-reassign
                                        props.innerRef.current = ref;
                                    }
                                    input.current = ref;
                                }}
```

Reply:
@s77rt ok, restored!

Reply:
@koko57 Please use `textInput` instead of `forwardedRef` e.g. in `onMouseDown`.

`forwardedRef` should only be referenced in the `ref` function

Reply:
@s77rt fixed!